opta/core/kubernetes.py

```diff
@@ -387,12 +394,102 @@ def _aws_set_kube_config(layer: "Layer") -> None:
     return
 
 
-def get_cluster_name(layer: "Layer") -> Optional[str]:
-    outputs = get_terraform_outputs(layer)
-    cluster_name = outputs.get("parent.k8s_cluster_name") or outputs.get(
-        "k8s_cluster_name"
-    )
-    return cluster_name
+def get_cluster_name(layer: "Layer") -> str:
+    return f"opta-{layer.root().name}"
+
+
+def does_cluster_exist(layer: "Layer") -> bool:
+    if layer.is_stateless_mode() is True:
+        if logger.isEnabledFor(DEBUG):
+            logger.debug(
+                "does_cluster_exist called in stateless mode, verify implementation. See stack trace below:"
+            )
+            traceback.print_stack()
```

Comment:
we can probably assume that cluster doesn't exist when running in stateless mode? False

Reply:
I don't understand-- should we just return false?

Reply:
yes return false.